examples/randomized-benchmark/DRAG-optimization/configuration.py

```diff
@@ -1,40 +1,72 @@
 import numpy as np
 
-gauss_len = 100
+gauss_len = 16
 gaussian_amp = 0.2
 
 
 def gauss(amplitude, mu, sigma, delf, length):
-    t = np.linspace(-length / 2, length / 2, length)
+    t = np.linspace(0, length, length)
     gauss_wave = amplitude * np.exp(-((t - mu) ** 2) / (2 * sigma ** 2))
-    # Detuning correction Eqn. (4) in Chen et al. PRL, 116, 020501 (2016)
-    gauss_wave = gauss_wave * np.exp(2 * np.pi * delf * t)
     return [float(x) for x in gauss_wave]
 
 
 def gauss_der(amplitude, mu, sigma, delf, length):
-    t = np.linspace(-length / 2, length / 2, length)
+    t = np.linspace(0, length, length)
     gauss_der_wave = (
-        amplitude * (-2 * (t - mu)) * np.exp(-((t - mu) ** 2) / (2 * sigma ** 2))
+        amplitude
+        * (-2 * 1e9 * (t - mu) / (2 * sigma ** 2))
+        * np.exp(-((t - mu) ** 2) / (2 * sigma ** 2))
     )
-    # Detuning correction Eqn. (4) in Chen et al. PRL, 116, 020501 (2016)
-    gauss_der_wave = gauss_der_wave * np.exp(2 * np.pi * delf * t)
     return [float(x) for x in gauss_der_wave]
 
 
+def gauss_det(amplitude, mu, sigma, delf, length):
```

Comment:
Too many functions with confusing names!
gauss -> Ok, simple
gauss_der -> I guess derivative, OK.
gauss_det ??
gauss_der_det ??

What are they?
I would change the names to be longer (and informtive), and while you're at it, add docstrings!

Reply:
Improved, thanks!